modules/imap/functions.php

```diff
@@ -251,10 +251,14 @@ function format_imap_message_list($msg_list, $output_module, $parent_list=false,
             $nofrom = ' nofrom';
         }
         $is_snoozed = !empty($msg['x_snoozed']) && hex2bin($msg['folder']) == 'Snoozed';
+        $is_scheduled = !empty($msg['x_schedule']) && hex2bin($msg['folder']) == 'Scheduled';
         if ($is_snoozed) {
-            $snooze_header = parse_snooze_header('X-Snoozed: '.$msg['x_snoozed']);
+            $snooze_header = parse_nexter_header('X-Snoozed: '.$msg['x_snoozed'], 'X-Snoozed');
```

Comment:
This is incoherent. You are getting the `snooze_header` though you seem to have replaced the correct function `parse_snooze_header` with `parse_nexter_header` which by its name does not intend to produce the expected output.